packages/fast-components-react-base/src/checkbox/checkbox.props.ts

```diff
@@ -36,9 +36,9 @@ export interface ICheckboxHandledProps {
     onChange?: CheckboxOnChange;
 
     /**
-     * The textual content
+     * The checkbox content
      */
-    text?: string;
+    children?: React.ReactNode | React.ReactNode[];
```

Comment:
Can you change this to `React.ReactNode`? It looks like the type `React.ReactNode` *includes* an array of `React.ReactNode`s so I think explicitly accepting an array here is unnecessary

Reply:
Sure